### PR TITLE
[stable/hazelcast-jet] Update default hazelcast-jet version

### PR DESCRIPTION
**Pull request by eminn**
Signed-off-by: Emin Demirci <emin@hazelcast.com>

#### What this PR does / why we need it:
Updates the default hazelcast-jet version to the latest one.

#### Which issue this PR fixes
N/A

#### Special notes for your reviewer:
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by eminn** [comment]
@davidkarlsen Could you please have a look at this ?

File: stable/hazelcast-jet/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-apiVersion: v1
```
Comment (maorfr) [inline stable/hazelcast-jet/Chart.yaml]: why was this line removed?

File: stable/hazelcast-jet/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
-appVersion: "0.7"
-description: Hazelcast Jet is an application embeddable, distributed computing engine built on top of Hazelcast In-Memory Data Grid (IMDG). With Hazelcast IMDG providing storage functionality, Hazelcast Jet performs parallel execution to enable data-intensive applications to operate in near real-time.
 name: hazelcast-jet
-version: 1.0.1
+version: 1.0.2
```
Comment (maorfr) [inline stable/hazelcast-jet/Chart.yaml]: lets bump to 1.1.0

**Review by maorfr — CHANGES_REQUESTED** [review]

File: stable/hazelcast-jet/Chart.yaml (hunk 1 above)
Comment (eminn) [reply stable/hazelcast-jet/Chart.yaml]: I thought that is not required but I've just double-checked this after your reminder, will add this again.

**Review by maorfr — APPROVED** [review]
